NDB.Covid19/NDB.Covid19.Test/Tests/Utils/LanguageTests.cs

```diff
@@ -71,32 +71,5 @@ public void CountryDetailsDTO_GetName_NotNullOrEmpty()
             Assert.NotNull(country.GetName());
             Assert.NotEmpty(country.GetName());
         }
-
-        [Theory]
-        [InlineData("nb", "Norge", "Noreg", "Norway", "Norge")]
-        [InlineData("en", "England", "England", "England", "England")]
-        [InlineData("nb", "Name_NB", "Name_NN", "Name_EN", "Name_NB")]
-        [InlineData("nn", "Name_NB", "Name_NN", "Name_EN", "Name_NN")]
-        [InlineData("en", "Name_NB", "Name_NN", "Name_EN", "Name_EN")]
-        [InlineData("pl", "Name_NB", "Name_NN", "Name_EN", "Name_NB")]
-        [InlineData("es", "Name_NB", "Name_NN", "Name_EN", "Name_NB")]
-        public void CountryDetailsDTO_GetName_ReturnsCorrect(string code, string nb, string nn, string en, string result)
```

Comment:
why removed?

Reply:
It will be obsolete when we merge changes from R2.0, since country translations are now handled in the backend